stdlib/2and3/modulefinder.pyi

```diff
@@ -22,6 +22,12 @@ class Module:  # undocumented
 class ModuleFinder:
 
     modules: Dict[str, Module]
+    path: Optional[List[str]] = ...  # undocumented
```

Comment:
`path` is initialized with `sys.path` if none is provided, so I think this can not be `Optional`.